TLM/TLM/UI/MainMenu/MainMenuWindow.cs

```diff
@@ -147,22 +147,22 @@ public class MainMenuWindow
         /// </summary>
```

Comment:
I can't add comment to untouched code but those fields are no longer used
```cs
private bool isStarted_;
//and
private MainMenuLayout menuLayout_;
```

Reply:
Removed, and this also makes MainMenuLayout class unused, removed too